#!/usr/bin/python3
def raise_exception_msg(message=""):
    raise NameError(message)
# copied with gratitude from another person's repo
